README.md

```diff
@@ -119,9 +119,12 @@ With 'up' and 'down', you can specify the option 'BOTH', which defaults to false
 By default, data migrations are added to the `db/data/` path.
 You can override this setting in `config/initializers/data_migrate.rb`
 
+Similarly, the default schema migration path is `db/migrate`, but you can override this by setting `schema_migrations_paths` (as below).
+
 ```ruby
 DataMigrate.configure do |config|
-  config.data_migrations_path = "db/awesomepath/"
+  config.data_migrations_path = 'db/awesomepath/'
+  config.schema_migrations_paths = ['db/migrate', 'awesome/db/migrate']
```

Comment:
hm, now that I read this, i'm a little confused. i thought the point of the PR is to allow multiple folders for data migrations. Schema migrations are handled by Rails. So did you mean to do `config.data_migrations_paths`? If both provided, can pick either one and print a warning message. What do you think?

Reply:
My purpose here is to make `db:migrate:with_data` handle multiple locations for schema migrations. This is something Rails does on it's own (https://www.rubydoc.info/docs/rails/4.1.7/ActiveRecord%2FSchema:migrations_paths) because the `migrations_paths` object is an array. But right now this gem hardcodes the one default schema migration location instead of respecting what Rails will use.

My thought was that this can allow users to determine which of their `migrations_paths` they care to use (or to set it to all of them). I'm happy to make multiple data migration paths possible, but that wasn't the intention with this PR.

Reply:
Oh, I see what you mean. Hm, that brings up an interesting point that this gem seems to bleed into something that it's not supposed to - doing schema migrations is not really it's job. However, allowing this task is valuable because it make sure that migrations (both data and schema) are run in specified order. (thinking out loud here :) )

I need stew on this a bit. How are you using this in your environment?

Reply:
I agree that "allowing this task is valuable because it make sure that migrations (both data and schema) are run in specified order." Since there isn't a separate gem that will maintain the order and run the schema migrations for us, it would be nice to allow this.

We have components/engines that have their own schema migrations, but not their own databases. These schema migrations are then run within the primary database.

Reply:
ok, you convinced me. so now, next question, if you have these schema migrations in different components, wouldn't you configure rails to know about all these locations where db migration files are stored? basically, why doesn't original `Rails.application.config.paths["db/migrate"]` work?

Reply:
we are configuring it with something along the lines of `Rails.configuration.database_configuration['common_migration_paths']['migrations_paths']`. which means in our case, `Rails.application.config.paths["db/migrate"].to_a` just returns `["<Rails.root>/db/migrate"]`